composer.json

```diff
@@ -45,7 +45,8 @@
             "Rector\\Doctrine\\": "packages/Doctrine/src",
             "Rector\\Utils\\": "packages/Utils/src",
             "Rector\\YamlRector\\": "packages/YamlRector/src"
-        }
+        },
+        "classmap": ["custom"]
```

Comment:
Why?

Reply:
So composer can load `custom/MergeIsCandidateRector.php`.

I'll move it to `/examples` in the end, so we have complex code to show people.